bin/main.rb

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env ruby
```

Comment:
put this line: **`# frozen_string_literal: true`** on top of line 1

Reply:
we have eliminated the global variables, no longer needed thanks :)